vcf_annotation_tools/vcf_readcount_annotator.py

```diff
@@ -69,6 +75,22 @@ def parse_bam_readcount_file(args):
                     coverage[(chromosome,position,reference_base)] = parsed_brct
     return coverage
 
+def has_snv(entry):
+    for alt in entry.ALT:
+        alt = alt.serialize()
+        ref = entry.REF
+        if len(ref) == len(alt):
```

Comment:
This is perfectly clear as is so perhaps not worth bothering with, but it looks like `has_snv` and `has_indel` differ only in a single line (the condition), they could probably be refactored. Though, its certainly possible the resulting indirection would be less clear than this version. Feel free to take or leave the suggestion :)